src/endpoints/network/network.service.ts

```diff
@@ -201,4 +166,61 @@ export class NetworkService {
   async getValidatorStatistics() {
     return await this.gatewayService.get('validator/statistics');
   }
+
+  async getApr(): Promise<{ apr: number, topUpApr: number, baseApr: number }> {
+    let stats = await this.getStats();
+    let config = await this.getNetworkConfig();
+    let stake = await this.stakeService.getGlobalStake();
+    const { account: { balance: stakedBalance } } = await this.gatewayService.get(`address/${this.apiConfigService.getAuctionContractAddress()}`);
+    let [ activeStake ] = await this.vmQueryService.vmQuery(
+      this.apiConfigService.getDelegationContractAddress(),
+      'getTotalActiveStake',
+    );
+    activeStake = this.numberDecode(activeStake);
+
+    let elrondConfig = {
```

Comment:
Shall we get this from an external config file? I assume they won't change too often (probably never) but you never know:))